applications/endoscopy_tool_tracking/python/CMakeLists.txt

```
@@ -63,3 +63,32 @@ if(BUILD_TESTING)
   )
 
 endif()
+
+# Install application and dependencies into the install/ directory for packaging
+install(
+  FILES ${CMAKE_CURRENT_SOURCE_DIR}/endoscopy_tool_tracking.py
+  DESTINATION "${CMAKE_SOURCE_DIR}/install/endoscopy_tool_tracking_python"
+  COMPONENT "holohub-apps"
+)
+
+file(READ "${CMAKE_CURRENT_SOURCE_DIR}/endoscopy_tool_tracking.yaml" APP_YAML_FILE)
+string(REPLACE "gxf_extensions/lstm_tensor_rt_inference/" "" APP_YAML_FILE "${APP_YAML_FILE}")
+file(WRITE "${CMAKE_SOURCE_DIR}/install/endoscopy_tool_tracking_python/endoscopy_tool_tracking.yaml" "${APP_YAML_FILE}")
```

Best to do this in two stages:
1. Write files generated at config time to the build directory, to avoid polluting sources in git
2. Reference the installation folder only via `install` in order to respect a user `--prefix` argument to `cmake install`

```suggestion
file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/packaging/endoscopy_tool_tracking.yaml" "${APP_YAML_FILE}")
install(
  FILE endoscopy_tool_tracking.yaml
  DESTINATION endoscopy_tool_tracking_python
  COMPONENT "holohub-apps"
)
```